zproject/db/backends/postgresql/base.py

```diff
@@ -0,0 +1,11 @@
+from django.db.backends.postgresql import base
+
+if False:
+    from typing import Any
+
+class DatabaseWrapper(base.DatabaseWrapper):
+    def __init__(self, *args, **kwargs):
+        # type: (*Any, **Any) -> None
+        super(DatabaseWrapper, self).__init__(*args, **kwargs)
+        # Always log queries and track times.
+        self.force_debug_cursor = True
```

Comment:
So I guess this differs from our previous implementation in that it logs the actual queries, not just the counts and time consumed, right?  

Can you check that the `write_log_line` function  in `zerver/middleware.py` handles this correctly?